### PR TITLE
Digibase integration

### PR DESCRIPTION
**Pull request by jfarbowitz**

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -370,6 +371,18 @@ lookup_qctoolsxml(){
...
 }
 
+lookup_db_choice(){
+    case "${1}" in
+        "Yes")
+            ;;
+        "No")
+            ;;
+        Q) exit 0 ;;
+        q) exit 0 ;;
```
Comment (retokromer) [inline vrecord]: Either `Q|q) exit 0 ;;` or `[Qq]) exit 0 ;;` would be one line shorter

**Comment by retokromer** [comment]
What is the status of this?

**Comment by retokromer** [comment]
Ping!

**Comment by dericed** [comment]
I recommend closing for the integration with digibase and superseding digibase's use with something similar to @privatezero's db integration in ltopers and mm. However there are non-digibase changes mixed in here which should be reviewed.

**Comment by retokromer** [comment]
The `audiomap` has also been changed in the master.

**Pull request closed without merging**